draft-ietf-oauth-transaction-tokens.md

```diff
@@ -433,6 +437,19 @@ grant_type=urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Atoken-exchange
 ~~~
 {: #figtxtokenrequest title="Example: Txn-Token Request"}
 
+### Subject Token Types
+The `subject_token_type` parameter in the Txn-Token Request MAY have one of the following values:
+
+rfc9068
+: The `subject_token` is a JWT format OAuth 2.0 Access Token as defined in {{RFC9068}}
+
+id-token
+: The `subject_token` is an OpenID Connect {{OpenIdConnect}} ID Token
+
+self-signed
+: The `subject_token` is a self-signed JWT issued by the requesting workload
```

Comment:
Other subject token types define the semantics of the token, and I think we should do that too. Maybe say (if this is true): 

> The `subject_token` is a self-signed OAuth access token issued by the requesting workload

Reply:
It's really not an OAuth access token though (acknowledging that what actually constitutes an OAuth access token isn't totally clear but calling this an access token would be a stretch and unnecessarily confusing).

Reply:
@bc-pi OK, but I guess we should qualify this JWT to get interoperability. Should we say what claims are REQUIRED?

Reply:
I don't know... I was only trying to politely point out that this thing is not an OAuth access token.